bin/run_workflow.py

```diff
@@ -103,6 +103,7 @@ def find_workflow(workflows, name):
     return candidates
 
 
+# FIXME: All these use old interface and it's horrible. I'm scared.
```

Comment:
This module is capable of finding a workflow (based on a filename) inside a given directory, bringing up an object based on it and then running it, optionally under a profiler.

This is handy when wearing a sysadmin hat, but crude and unnecessary for a reasonably designed python application.

I could make it work with the latest changes in this branch (and I should if we want to not break workflow) or kill it.